translationStatusScript.js

```diff
@@ -0,0 +1,109 @@
+const fs = require('node:fs')
+
+const contentBase = `---
+layout: page
+title: Status Translate
+menu: resources
+lang: en
+redirect_from: "/resources/status-translate.html"
+---
+
+# Status Translate
+
+This page shows the status of the translation of the documentation into different languages.
+
+| en |`
+
+let content = contentBase
+
+const languageBase = 'en'
+const verifyLanguage = ['pt-br', 'es', 'de', 'fr', 'it', 'ja', 'ko', 'ru', 'tr', 'th', 'sk', 'uz', 'zh-cn', 'zh-tw']
+
+const statusTranslate = []
+statusTranslate[`${languageBase}`] = []
+
+verifyLanguage.forEach((language) => {
+  content = content + ` ${language} |`
+  statusTranslate[`${language}`] = []
+})
+
+content = content + '\n |---|'
+
+verifyLanguage.forEach(() => {
+  content = content + '---|'
+})
+
+content = content + '\n'
+
+const ignoreFiles = ['resources/status-translate']
+
+const languageBaseFiles = ['guide', 'resources', 'resources/middleware', 'starter', 'advanced', '3x', '4x', '5x']
```

Comment:
It is not necessary to keep checking if the API versions are translated, and I don't think it will be necessary to keep looking at 'resources/middleware' either, as it is an external resource that is extracted by running a script.

Reply:
in the language (th) there are translated 'resources/middleware'

https://deploy-preview-1586--expressjscom-preview.netlify.app/en/resources/status-translate

![Captura de Tela 2024-09-01 às 22 09 38](https://github.com/user-attachments/assets/7e7a839f-cbb5-415f-8c09-3215dcc3c310)

the API page has been translated into other languages